torchaudio/compliance/kaldi.py

```diff
@@ -826,21 +770,4 @@ def resample_waveform(waveform: Tensor,
     Returns:
         Tensor: The waveform at the new frequency
     """
-    assert waveform.dim() == 2
```

Comment:
This PR is also adding batching to kaldi compliance interface. Please add that note in the description of the PR so we know this feature has been added as part of this PR.